### DISCUSSION TITLE
localhost domains

### DISCUSSION THREAD

**Discussion by ThomasRedstone** [wardenenv/warden#667] [Ideas]
Hello, I've usually stuck to using `*.localhost` for my local testing; all domains automatically resolve to 127.0.0.1, and it doesn't require any changes to your DNS server configuration.

Are there advantages to using `.test` instead, or could it be worth considering using `.localhost`?

**Comment by navarr** [comment]
I believe the automatic binding of *.localhost to 127.0.0.1 is not in all Operating Systems.  *.test is likewise reserved for testing software, and is rarely bound by the host OS, which means there should be few issues when we attempt to bind it.

**Reply by ThomasRedstone** [reply]
Yes, you're quite right, but it does work automatically on Windows in Edge, Chrome and Firefox, so, it can be pretty handy for use in browsers.
